app/controllers/home_controller.rb

```diff
@@ -1,4 +1,33 @@
 class HomeController < ApplicationController
   def index
+    @hotels = Hotel.all
+  end
+
+  def contact_new
+    @contact_form = if current_user
+      ContactForm.new(
+        email: current_user.email
+      )
+    else
+      ContactForm.new
+    end
+  end
+
+  def contact_create
+    @contact_form = ContactForm.new(contact_form_params)
+    if @contact_form.save
```

Comment:
if not>? in both cases it will render in 1 case with notification in other without notification i think there should be one in case failed